apps/nrm/behaviours/casework-submission.js

```diff
@@ -33,6 +31,11 @@ module.exports = conf => {
         const caseWorkPayload = appConfig.writeToCasework ? config.prepare(req.sessionModel.toJSON()) :
           { info: 'No submission was made to icasework' };
 
+        const externalID = req.sessionModel.get('externalID') ?
+          req.sessionModel.get('externalID') : caseWorkPayload.ExternalId;
```

Comment:
```suggestion
          const externalID = req.sessionModel.get('externalID')  || caseWorkPayload.ExternalId;
```

Reply:
This is cool, didn't know you can check an undefined like this. Changed to this.